bookwyrm/templates/snippets/book_titleby.html

```diff
@@ -9,12 +9,19 @@
 
 {% if book.authors.exists %}
 {% blocktrans trimmed with path=book.local_path title=book|book_title %}
-<a href="{{ path }}">{{ title }}</a> by
+<a href="{{ path }}">{{ title }}
+</a> 
+by
 {% endblocktrans %}&nbsp;{% include 'snippets/authors.html' with book=book limit=3 %}
 
 {% else %}
 <a href="{{ book.local_path }}">{{ book|book_title }}</a>
 {% endif %}
 
+{% if book.series %}
+({{book.series}}{%if book.series_number %}, #{{book.series_number}}{% endif %})
+{% endif %}
```

Comment:
Maybe we want to put this piece in a separate snippet that we include here and in all the headers? That way there is a single point of definition for how the book series is rendered.